posts/a-beginners-guide-to-influxdb.md

```diff
@@ -0,0 +1,109 @@
+---
+authors:
+- Ketki Deshpande
+tags:
+- Data
+date: 
+title: "A beginner’s guide to InfluxDB - A time series database"
+image: 
+---
+
+A time series database (TSDB) is specially made for data that can be evaluated as a time-series. Data like sensor outputs, market trends, cpu utilization or application performance metrics are some of the examples of data which can be best analyzed over a time period (a time-series). Most popular types of databases (SQL or No-SQL databases) are not optimized for handling large scale scans, summarization or aggregation of time-series data. Time-series databases have built-in functionality to work with aggregation, down-sampling, data lifecycle management, summarization and so on of time-series data. Let’s look at one of the popular time-series databases, InfluxDB. InfluxDB uses line protocol for sending and storing the data.
```

Comment:
Time-series databases have built-in functionality to work with aggregation, down-sampling, data lifecycle management, summarization, etc.